permits/models.py

```diff
@@ -898,6 +900,7 @@ class WorksObjectProperty(models.Model):
     # The choices are sorted according to their values
     INPUT_TYPE_CHOICES = (
         (INPUT_TYPE_ADDRESS, _("Adresse")),
+        (INPUT_TYPE_PARCEL, _("Parcelle")),
```

Comment:
Should be between "Nombre" and "Texte" (the choices are sorted according to their values)